plugin/types.go

```diff
@@ -195,7 +195,8 @@ type Readable interface {
 	Open(context.Context) (SizedReader, error)
 }
 
-// Deletable is an entry that can be deleted.
+// Deletable is an entry that can be deleted. Its semantics are described
+// in https://puppetlabs.github.io/wash/docs/external-plugins#delete
```

Comment:
I think people should be able to figure out what the return values mean from the external plugin docs.

Reply:
Someone - including our future selves - writing core plugins might not think to go look there.

Reply:
The core plugin documentation should largely stand on its own.

Reply:
Ah, I guess you link to it. But it feels weird to me for the core plugin documentation to link out to an external page just to explain what the function does and returns.